python/tskit/.ipynb_checkpoints/trees-checkpoint.py

```diff
@@ -6863,12 +6865,14 @@ def _forward_extend(self, forwards=True):
                 edges_out.append(tk)
```

Comment:
Doh! Good (and, tricky) catch.

Would things be simpler before if we did
```
edges_out.append(O[tk])
```
here, instead of changing the various `tk` to `O[tk]` below? (And, same for `tj`.)

Reply:
my vote is yes!